src/main/include/subsystems/Climber.hpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020 FRC Team 3512. All Rights Reserved.
+
+#pragma once
+
+#include <frc/Joystick.h>
+#include <frc/PowerDistributionPanel.h>
+#include <frc/Solenoid.h>
+#include <rev/CANSparkMax.h>
+
+#include "Constants.hpp"
+#include "communications/PublishNode.hpp"
+#include "subsystems/SubsystemBase.hpp"
+
+namespace frc3512 {
+
+class Climber : public SubsystemBase, public PublishNode {
+public:
+    Climber();
+
+    void SetTransverser(double speed);
+
+    void SetElevator(double speed);
+
+    void ProcessMessage(const ButtonPacket& message);
+
+private:
+    rev::CANSparkMax m_armL{frc3512::Constants::Climber::kArmPortL,
+                            rev::CANSparkMax::MotorType::kBrushless};
+    rev::CANSparkMax m_armR{kArmPortR, rev::CANSparkMax::MotorType::kBrushless};
```

Comment:
Call them m_armLeft and m_armRight. Characters are cheap.

Reply:
Change applied.